spec/support/spree/config_helper.rb

```diff
@@ -0,0 +1,18 @@
+module Spree
+  module ConfigHelper
+    def with_spree_config(hash)
+      stored = config_slice(hash.keys)
+      Config.set(hash)
+      yield
+      Config.set(stored)
```

Comment:
With this line, Would we restore only the value of the keys we modified earlier or does `.set` behave like a merge? It looks like the former to me and this could affect some tests.